server/database/db_build.sql

```diff
@@ -0,0 +1,132 @@
+BEGIN;
+  DROP TABLE IF EXISTS places, guides, users, Arabic_words
```

Comment:
Cases of names should be kept consistent e.g using lowercase instead of uppercase.